other/activity_selection.py

```diff
@@ -0,0 +1,38 @@
+"""The following implementation assumes that the activities 
+are already sorted according to their finish time"""
+  
+"""Prints a maximum set of activities that can be done by a 
+single person, one at a time"""
+# n --> Total number of activities 
+# s[]--> An array that contains start time of all activities 
+# f[] --> An array that contains finish time of all activities 
+  
+def printMaxActivities(s, f ): 
```

Comment:
Great progress!  Please add this doctest to the printMaxActivities() function and once it is passing, I think we can land this one.  See CONTRIBUTING.md for details about doctests.
```
    """
    >>> start = [1, 3, 0, 5, 8, 5] 
    >>> finish = [2, 4, 6, 7, 9, 9] 
    >>> printMaxActivities(start, finish)
    The following activities are selected
    0 1 3 4
    """
```